Models/NHES/Systems/PrimaryHeatSystem/HTGR/package.order

```diff
@@ -1,9 +1,3 @@
-Examples
-SubSystem_Dummy
-CS_Dummy
-ED_Dummy
-Components
-Data
-BaseClasses
-CS_Basic
-CS_Rankine
+Brayton_Systems
+HTGR_Rankine_Mikk_In_Progress
```

Comment:
Not sure this is the write package.order given that we don't include HTGR_Rankine_Mikk_In_Progress